src/javascript/_common/base/gtm.js

```diff
@@ -60,7 +60,7 @@ const GTM = (() => {
             bom_country_abbrev: get_settings.country_code,
             bom_email         : get_settings.email,
             url               : window.location.href,
-            bom_today         : Math.floor(Date.now() / 1000),
+            bom_today         : window.time,
```

Comment:
`window.time` is`undefined` for me in chrome and firefox. Did we create the value ourself?

Reply:
you would have to wait for the response of time before expecting the value.